11-payment-encoding.md

```diff
@@ -190,7 +191,7 @@ A writer:
   - if `9` contains non-zero bits:
     - SHOULD use the minimum `data_length` possible.
   - otherwise:
-    - MUST omit the `9` field altogether.
+    - MUST omit the `9` and `s` field altogether.
```

Comment:
I think you can remove that change. A requirement a bit further down already specifies that: "MUST set an s field if and only if the payment_secret feature is set."